src/middlewares/getOneValidation.spec.js

```diff
@@ -0,0 +1,40 @@
+import sinon from 'sinon';
+import { getOneValidation, schema } from './getOneValidation';
+
+
+
+describe('middlewares - getOneValidation()', () => {
+    const sandbox = sinon.createSandbox();
+    let req, res, next, validatingStub;
+    beforeEach(() => {
+        validatingStub = sandbox.stub(schema, 'validate');
+        req = {
+            query: {
+                name: null,
+                street: null,
+                postalCode: null,
+            },
+        };
+        next = sandbox.stub();
+    });
+    afterEach(() => {
+        sandbox.restore();
+    });
+    it('Missed one of data in query - returns an error', () => {
+        const validationResult = { error: { details: [{ message: 'Validation Error' }] } };
+        validatingStub.returns(validationResult);
```

Comment:
can we make it work without stubbing `schema`? 
the test should see the same error message users would see